kong/init.lua

```diff
@@ -781,7 +783,80 @@ function Kong.init()
 end
 
 
+local maybe_init_emmy_debugger
```

Comment:
Consider putting this debugger initialization code inside it's own file. `init.lua` is already too bloated today.